Bug: Dualtor ip route verification fails as command not found

### Is it platform specific

generic

### Importance or Severity

High

### Description of the bug

#22956 changed the route verification command from "ip route show" to "show ip route", as the use of the latter allows for a netlink bug to be bypassed on the SONiC DUT.  This command does not exist on the ptf, and so fails.

### Steps to Reproduce

Run any of the following dualtor tests:

- test_bgp_block_loopback1
- test_orchagent_slb

### Actual Behavior and Expected Behavior

Route verificaion should not fail with "command not found" on the ptf.

### Relevant log output

```shell
0:00:00.002023&#10;msg = non-zero return code&#10;invocation = {'module_args': {'_raw_params': 'show ip route', '_uses_shell': True, 'expand_argument_vars': True, 'stdin_add_newline': True, 'strip_empty_ends': True, 'argv': None, 'chdir': None, 'executable': None, 'creates': None, 'removes': None, 'stdin': None}}&#10;_ansible_no_log = False&#10;stdout =&#10;stderr =&#10;/bin/sh: 1: show: not found&quot;">ptfhost = &lt;tests.common.devices.ptf.PTFHost object at 0x7f7bf0109a90&gt;, upper_tor_host = &lt;MultiAsicSonicHost ldp203&gt;, lower_tor_host = &lt;MultiAsicSonicHost ldp205&gt;
&gt;           ptfhost.shell("show ip route")
self = &lt;tests.common.devices.ptf.PTFHost object at 0x7f7bf0109a90&gt;, module_args = ['show ip route'], complex_args = {}
function_name = 'setup_interfaces', lines = ['        ptfhost.shell("show ip route")\n'], index = 0, verbose = True, module_ignore_errors = False, module_async = False
E           cmd = show ip route
E           invocation = {'module_args': {'_raw_params': 'show ip route', '_uses_shell': True, 'expand_argument_vars': True, 'stdin_add_newline': True, 'strip_empty_ends': True, 'argv': None, 'chdir': None, 'executable': None, 'creates': None, 'removes': None, 'stdin': None}}
lines      = ['        ptfhost.shell("show ip route")\n']
module_args = ['show ip route']
```

### Output of `show version`

```shell

```

### Attach files (if any)

_No response_

--- a/tests/dualtor/conftest.py
+++ b/tests/dualtor/conftest.py
@@ -273,7 +273,7 @@ def setup_interfaces(ptfhost, upper_tor_host, lower_tor_host, tbinfo, test_devic
     finally:
         upper_tor_host.shell("show arp")
         lower_tor_host.shell("show arp")
-        ptfhost.shell("show ip route")
+        ptfhost.shell("ip route show")
         for conn in list(connections.values()):
             ptfhost.shell("ifconfig %s 0.0.0.0" % conn["neighbor_intf"], module_ignore_errors=True)
             ptfhost.shell("ip route del %s" % conn["local_addr"], module_ignore_errors=True)

